pkgs/build-support/fetchpatch/default.nix

```diff
@@ -15,6 +15,10 @@ in
 , excludes ? []
 , includes ? []
 , revert ? false
+, throwBool ? false
```

Comment:
not sure that communicates well what this argument does. would go for something like `validIf ? true` to make that part of a fetchpatch call easier to understand, but that doesn't really explain the "throw unless validIf" either. `throwUnless` maybe? not sure.